src/Common/src/CoreLib/System/Math.cs

```diff
@@ -22,9 +22,9 @@ namespace System
 {
     public static partial class Math
     {
-        public const double E = 2.7182818284590451;
```

Comment:
You shouldn't need to modify the files in `src\Common`, they will flow over naturally after the CoreCLR side is merged.

Reply:
Nevermind, GitHub put me on an outdated commit.